### ISSUE TITLE
You shouldn't be able to export demo games

### ISSUE THREAD

**Issue by extremestrategy** [TheGameCreators/GameGuruRepo#5289]
So the problem is in the title, you can currently export to standalone for the demo games (ie trapped, island showdown, etc), here is a screenshot of an exported demo game,

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/114613272/b41a95fb-39f9-4c1b-a7b8-9b5fa32d3c38)

Max should have a popup or something that stops this from happening as some people would exploit it, and try and claim the games as "there creations" which could damage the GGM reputation.

**Comment by 3coms** [comment]
A bit difficult to do; to do so, Lee should protect demos against saving, rather than standalone.
Imagine someone runs _Island Showdown_ demo and places some meshes here and there, then saves it with a different name, and then makes a standalone with it.
To get it, no one should be able to save demos, just use them as a reference to learn the how-to.

**Comment by extremestrategy** [comment]
True, so disable the saving, and the save standalone for the demo maps/projects, good idea 3com.

**Comment by Blood-Moon-Interactive** [comment]
I agree that these should have some sort of special control around them. The tricky part is ideally we want them to be editable so that people can learn from them but not exportable or something that can be saved and exploited in that way. I'd like to eventually contribute a demo for learning purposes and know that it won't show up one day on steam.

**Comment by extremestrategy** [comment]
Maybe allow saving, but not a save as function, or forcing them to be saved in a format that only works with demo games, I don't know, just throwing ideas.

**Comment by Kraven3D** [comment]
I requested this here some time ago and was told it was, "planned to a certain extent":
https://github.com/TheGameCreators/GameGuruRepo/issues/3494

Honestly surprised we haven't seen any demo games pop up on Steam yet, as that was almost 18 months ago!

**Comment by extremestrategy** [comment]
That's exactly my concern, demo games on Steam or Itch.io, and that coming back at Max, and I think the only reason it hasn't happened is no one thought to try it, or they didn't read the manual well enough to know how to export them.
